### PR TITLE
TEDEFO-1065 export of various files with the new labels.

### PR DESCRIPTION
**Pull request by rouschr**
This allows to compare the new vs old labels.
Let me know if there are too many files.
I could restrict this to just the:
-  translations
-  codelists 
-  view-templates
-  some of the index mapping files

### CODE REVIEW

**Comment by rouschr** [comment]
Let me know if there are too many files.
I could restrict this to just the translations, codelists and some of the index mapping files.

**Review by bertrand-lorentz — CHANGES_REQUESTED** [review]
The translations for "business-term" do not have the expected filename. Currently it is:
translations/business_term_en.xml

It should be:
translations/business-term_en.xml

I have a fix for this in MDC, I'll submit it shortly.

**Comment by rouschr** [comment]
I reexported the translations with the correct filenames after merging the MDC fix.

File: translations/expression_en.xml [hunk 1]
```diff
@@ -1,384 +1,388 @@
...
-<entry key="expression|description|90">the value chosen for BT-142-LotResult is 'At least one winner was chosen'</entry>
-<entry key="expression|description|91">BT-3201-Tender is not empty</entry>
-<entry key="expression|description|93">the value chosen for BT-727-Lot is 'Anywhere in the given country' or BT-727-Lot is empty</entry>
-<entry key="expression|description|94">the value chosen for BT-727-Part is 'Anywhere in the given country' or BT-727-Part is empty</entry>
-<entry key="expression|description|95">the value chosen for BT-727-Procedure is 'Anywhere in the given country' or BT-727-Procedure is empty</entry>
-<entry key="expression|description|96">BT-541-Lot is present and BT-5422-Lot is not present and BT-5423-Lot is not present</entry>
-<entry key="expression|description|97">BT-541-LotsGroup is present and BT-5422-LotsGroup is not present and BT-5423-LotsGroup is not present</entry>
+<comment>expression in English. File generated from metadata database version 0.5.169 created on the 2022-07-27T15:57:08</comment>
+<entry key="expression|text|1">the value chosen for the indicator of BT-740-Procedure-Buyer is 'NO'</entry>
```
Comment (bertrand-lorentz) [inline translations/expression_en.xml:5]: I'm not sure we had decide to go from `expression|description|...` to `expression|text|...`
Please confirm that it was intended.

**Review by bertrand-lorentz — COMMENTED** [review]
It would have been better to have one commit per type of file, as some files have other changes included. For example fields.json has the move of the XML structure.

**Review by rousso — APPROVED** [review]

**Review by rousso — CHANGES_REQUESTED** [review]
I agree with Bertrand. There are important files missing here wheres some files are not really changed.
The codelists for example are not changed essentially.
The EFX view templates are essential and need to be brought to the develop branch of the SDK.
I propose to close this PR without merging and to separate PRs per file type so that we can have more control as Bertrand suggests.

File: translations/expression_en.xml (hunk 1 above)
Comment (rousso) [reply translations/expression_en.xml:5]: Indeed we did not decide to go from expression description to expression text. 
We should remain with expression|description.

Comment (rouschr) [reply translations/expression_en.xml:5]: I do not see any of my SQL which could have done this:
UPDATE label set asset_type = 'expression' where asset_type = 'condition';
I did not change the description to text.

**Pull request closed without merging**